extensions/sql-migration/src/api/utils.ts

```diff
@@ -782,8 +782,14 @@ export async function createPowershellscriptContentContainer(view: azdata.ModelV
 	}).component();
 
 	const scriptpath = path.join(__dirname, '../scripts/SHIR-auto-configuration.ps1');
+	const scriptpath2 = path.join(__dirname, '../../scripts/SHIR-auto-configuration.ps1');
 
-	const scriptContent = await fs.readFile(scriptpath);
+	let scriptContent;
```

Comment:
since we are not logging any error , instead of try catch can we use if-else to check file exists and then read file.

Reply:
One of the possible error is that path exists or not but other can be there as well like insufficient permissions, and I/O errors. This blade is failing for some folks but not for all. I find it better to put in try catch block, rather than just checking one possible error.